replifier/README.md

```diff
@@ -0,0 +1,97 @@
+# replifier
+
+The "replifier" (replicator–verifier) is responsible for:
+
+*   Creating copies of our bags in permanent storage
+*   Verifying that a bag is "correct" -- checksums match the files, no unreferenced files were included, fetch.txt URIs are consistent, and so on
+
+
+
+## How data moves through the storage service
+
+![](data_movements.png)
+
+1.  The user uploads a bag to the **user uploads bucket**.
+
+2.  The bag unpacker unpacks the user-uploaded bag into the **unpacked bags bucket**.
+
+    We want it inside our AWS account as soon as possible (so it can't be deleted under our feet).
```

Comment:
stupid question: does that mean that the `user uploads bucket` is outside our AWS account?

Reply:
Not stupid – right now it’s inside our AWS estate but not the same account (we have upload buckets in the workflow/digitisation accounts), but in theory it could be anywhere. I'll clarify.

Reply:
Ignore it, I was reading the non rendered version so wasn't seeing the picture with the clear boundary of our account